ensuring cross section is specified before submitting jobs

andreh7 opened the pull request:
with this patch, fggRunJobs.py will exit if a cross section for a selected process is missing in the cross sections file (to avoid have the jobs crash after submission and then resubmit them multiple times).

molmedon commented:
Hi all is there a reason why we don’t have a SingleMuon and DoubleMuon entry here ?

https://github.com/cms-analysis/flashgg/blob/master/MetaData/data/cross_sections.json#L14


On Apr 22, 2016, at 12:57 PM, andreh7 <notifications@github.com<mailto:notifications@github.com>> wrote:


with this patch, fggRunJobs.py will exit if a cross section for a selected process is missing in the cross sections file (to avoid have the jobs crash after submission and then resubmit them multiple times).

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/cms-analysis/flashgg/pull/624

Commit Summary

  *   class SamplesManager: changed default cross section from 0 to None so we can detect if the cross section was not specified in the json file
  *   class JobsManager: now checking whether all cross sections are present before submitting the jobs (which would then crash later because the cross section is not found)
  *   update on checking cross sections in class JobsManager before submitting jobs: fixed typo

File Changes

  *   M MetaData/python/jobs_utils.py<https://github.com/cms-analysis/flashgg/pull/624/files#diff-0> (66)
  *   M MetaData/python/samples_utils.py<https://github.com/cms-analysis/flashgg/pull/624/files#diff-1> (2)

Patch Links:

  *   https://github.com/cms-analysis/flashgg/pull/624.patch
  *   https://github.com/cms-analysis/flashgg/pull/624.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/cms-analysis/flashgg/pull/624>

andreh7 commented:
no, there is no reason apart from nobody asking for it so far. I'll update
this branch with corresponding entries.

On 22 April 2016 at 23:02, molmedon <notifications@github.com> wrote:

> Hi all is there a reason why we don’t have a SingleMuon and DoubleMuon
> entry here ?
>
>
> https://github.com/cms-analysis/flashgg/blob/master/MetaData/data/cross_sections.json#L14
>
>
> On Apr 22, 2016, at 12:57 PM, andreh7 <notifications@github.com<mailto:
> notifications@github.com>> wrote:
>
>
> with this patch, fggRunJobs.py will exit if a cross section for a selected
> process is missing in the cross sections file (to avoid have the jobs crash
> after submission and then resubmit them multiple times).
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/cms-analysis/flashgg/pull/624
>
> Commit Summary
>
> * class SamplesManager: changed default cross section from 0 to None so we
> can detect if the cross section was not specified in the json file
> * class JobsManager: now checking whether all cross sections are present
> before submitting the jobs (which would then crash later because the cross
> section is not found)
> * update on checking cross sections in class JobsManager before submitting
> jobs: fixed typo
>
> File Changes
>
> * M MetaData/python/jobs_utils.py<
> https://github.com/cms-analysis/flashgg/pull/624/files#diff-0> (66)
> * M MetaData/python/samples_utils.py<
> https://github.com/cms-analysis/flashgg/pull/624/files#diff-1> (2)
>
> Patch Links:
>
> * https://github.com/cms-analysis/flashgg/pull/624.patch
> * https://github.com/cms-analysis/flashgg/pull/624.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub<
> https://github.com/cms-analysis/flashgg/pull/624>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-analysis/flashgg/pull/624#issuecomment-213587992>
>

molmedon commented:
Thank you!
On Apr 23, 2016, at 12:24 AM, andreh7 <notifications@github.com<mailto:notifications@github.com>> wrote:

no, there is no reason apart from nobody asking for it so far. I'll update
this branch with corresponding entries.

On 22 April 2016 at 23:02, molmedon <notifications@github.com<mailto:notifications@github.com>> wrote:

> Hi all is there a reason why we don’t have a SingleMuon and DoubleMuon
> entry here ?
>
>
> https://github.com/cms-analysis/flashgg/blob/master/MetaData/data/cross_sections.json#L14
>
>
> On Apr 22, 2016, at 12:57 PM, andreh7 <notifications@github.com<mailto:notifications@github.com><mailto:
> notifications@github.com<mailto:notifications@github.com>>> wrote:
>
>
> with this patch, fggRunJobs.py will exit if a cross section for a selected
> process is missing in the cross sections file (to avoid have the jobs crash
> after submission and then resubmit them multiple times).
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/cms-analysis/flashgg/pull/624
>
> Commit Summary
>
> * class SamplesManager: changed default cross section from 0 to None so we
> can detect if the cross section was not specified in the json file
> * class JobsManager: now checking whether all cross sections are present
> before submitting the jobs (which would then crash later because the cross
> section is not found)
> * update on checking cross sections in class JobsManager before submitting
> jobs: fixed typo
>
> File Changes
>
> * M MetaData/python/jobs_utils.py<
> https://github.com/cms-analysis/flashgg/pull/624/files#diff-0> (66)
> * M MetaData/python/samples_utils.py<
> https://github.com/cms-analysis/flashgg/pull/624/files#diff-1> (2)
>
> Patch Links:
>
> * https://github.com/cms-analysis/flashgg/pull/624.patch
> * https://github.com/cms-analysis/flashgg/pull/624.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub<
> https://github.com/cms-analysis/flashgg/pull/624>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-analysis/flashgg/pull/624#issuecomment-213587992>
>

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/cms-analysis/flashgg/pull/624#issuecomment-213681001>
